src/ConcreteOperations/difference.jl

```diff
@@ -111,3 +113,40 @@ function difference(I1::IN, I2::IN)::Union{EmptySet{N}, IN, UnionSet{N, IN, IN}}
         end
     end
 end
+
+# ============================================
+# Set difference between hyperrectangular sets
+# =============================================
+
+"""
+    difference(X::AbstractHyperrectangle{N}, Y::AbstractHyperrectangle{N}) where {N}
+
+Return the set difference between the given hyperrectangular sets.
+
+The backslash symbol, `\\`, can be used as an alias.
```

Comment:
I would put this in the `### Notes` section, if at all (we do not note this for `intersection` etc.).